package.json

```diff
@@ -39,6 +39,7 @@
     "inquirer": "^4.0.0",
     "json-to-ast": "https://github.com/johnspackman/json-to-ast.git#editable-json",
     "node-fetch": "^1.7.3",
+    "qooxdoo-sdk": "^5.0.2",
```

Comment:
Wouldn't it have to be 6.0.0-alpha? Does 5.0.2 work with the new compiler, @johnspackman ?

Reply:
I can't remember whether the changes required (or all the changes, anyway) are in 5.0.2.  They are backward compatible with the generator though, so it should be possible to do it quite easily if it's not already there